package.json

```diff
@@ -52,6 +62,18 @@
         "key": "ctrl+alt+h",
         "mac": "ctrl+alt+h",
         "when": "editorTextFocus"
+      },
+      {
+        "command": "extension.dash.syntax",
+        "key": "ctrl+shift+h",
+        "mac": "ctrl+shift+h",
+        "when": "editorTextFocus"
+      },
+      {
+        "command": "extension.dash.customWithSyntax",
+        "key": "shift+alt+h",
+        "mac": "shift+cmd+h",
```

Comment:
`shift+cmd+h` is already used for replace files shortcut

![image](https://user-images.githubusercontent.com/1243921/27734336-c24d6d2c-5ddd-11e7-9279-d677e3cc149d.png)

I suggest we go for `ctrl+cmd+h`. What do you think?